lib/webauth.js

```diff
@@ -86,14 +92,19 @@ AuthResponse.prototype = {
       //or may know ahead of time due to set value or retrievable server config.
       handlerResult.expms = handler.sessionExpirationMS;
     }
+    //overall expiration when last auth expires
```

Comment:
so when handler A has expired but plugin B hasn't yet, then functions protected by A will stop working, but those protected  by B wont?

Reply:
Yes. So, we're leaving it up to the UI to make good UI alerts about expiration. I just wanted an overall expiration so that the session wouldn't exist forever.